src/ica_api.c

```diff
@@ -2761,6 +2767,9 @@ unsigned int ica_aes_ctr(const unsigned char *in_data, unsigned char *out_data,
 #ifdef ICA_FIPS
 	if (fips >> 1)
```

Comment:
Here we right-shift by one.  In line 2273 we use `fips` directly.  Is this desired?

Reply:
Yes, DES is disabled in FIPS mode, but 3DES not. So we can simply remove the check from the ica_des_ctr routine.